### ISSUE TITLE
export function

### ISSUE THREAD

**Issue by panavrin** [echo-lab/SensDat#17]
The export function not working. 

The export button should generate a CSV file (or more) whatever "visible" table at the moment (summary table or raw data).

**Comment by Andy1uu** [comment]
If you guys could give some more information about this issue, that would be greatly appreciated since this seems like the highest priority issue that I should be working on.

**Comment by maneshd** [comment]
The TL;DR: is that the "export" button should yield CSVs that look just like the data displayed in the table in SensDat. 

There's basically two parts:
1. Get the data for the currently-available table(s) (i.e., the main table, or a summary table). 
2. Figure out how to put the data into a CSV that's downloaded on the client. I think [this stack overflow post](https://stackoverflow.com/questions/14964035/how-to-export-javascript-array-info-to-csv-on-client-side) explains one way to do it.

Part 1 will require understanding the code a bit :) You can see which tab is currently active by checking [activeTab](https://github.com/echo-lab/SensDat/blob/fb11db233343ab87d6612ea62ca4ff9ae0728204/src/app-state.js#L67) in the appState. I suggest trying to read the code to understand how you can get the data you need from the DataTable and SummaryTable. Off the top of my head, the SummaryTable data is maybe a little more complicated/annoying, so you could start out by having export only work on the base DataTable, and then implement the SummaryTable after that.
